context/app/static/js/components/Routes/Routes.jsx

```diff
@@ -270,6 +271,14 @@ function Routes({ flaskData }) {
     throw Error('Intentional client-side-error');
   }
 
+  if (urlPath.startsWith('/genes/')) {
+    return (
+      <Route>
+        <Genes organ={organ} collection={entity} />
```

Comment:
```suggestion
        <Genes />
```
You updated the flask route to no longer pass the organ in flask data. I imagine collection/entity is also a vestige of the copied route. You also want to past the gene name in, but that could be done in a later PR if that's your preference.

Reply:
Completely missed that part!